<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dlg_IllegalDict</class>
 <widget class="QWidget" name="Dlg_IllegalDict">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>678</width>
    <height>477</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dlg_IllegalDict</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTreeWidget" name="treeWidget">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <attribute name="headerDefaultSectionSize">
      <number>150</number>
     </attribute>
     <column>
      <property name="text">
       <string>序号</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>违法名称</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>违法代码</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>协议代码</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btn_save">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>28</height>
        </size>
       </property>
       <property name="text">
        <string>保存</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_reset">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>28</height>
        </size>
       </property>
       <property name="text">
        <string>重置</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
